app/queries/my_facilities/follow_ups_query.rb

```diff
@@ -0,0 +1,32 @@
+class MyFacilities::FollowUpsQuery
+  # Wrap query method calls with the appropriate timezone in which the reports will be consumed
+  # This is probably the Rails.application.config.country[:time_zone]
+  # Example: `Time.use_zone('timezone string') { bp_control_query_object.cohort_registrations }`
+
+  include QuarterHelper
+  include MonthHelper
+  include DayHelper
+  include PeriodHelper
+
+  attr_reader :periods
+
+  def self.total_follow_ups(facilities)
```

Comment:
Even though this doesn't use all the initialization args, I think this should still be an instance method.

Reply:
Addressed in https://github.com/simpledotorg/simple-server/pull/843/commits/df92264398c30997e9fe4571b9ac938becce08ff.